Dockerfile

```
@@ -0,0 +1,6 @@
+FROM node:latest
```

stable 버전으로 고정하는게 안전할거같아요

---

```suggestion
FROM node:20.13.1
```
노드 공문보니 이게 stable인듯?

---

stable 버전으로 반영했습니당~